scrapy/http/response/__init__.py

```diff
@@ -110,65 +132,65 @@ def _set_body(self, body):
 
     body = property(_get_body, obsolete_setter(_set_body, "body"))
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"<{self.status} {self.url}>"
 
-    def copy(self):
+    def copy(self) -> Response:
         """Return a copy of this Response"""
         return self.replace()
 
-    def replace(self, *args, **kwargs):
+    def replace(self, *args: Any, **kwargs: Any) -> Response:
```

Comment:
I think this could cause some false positives with the type checking, when using response subclasses.

Reply:
Do you mean false negatives?

Reply:
False positives in the user code, i.e. mypy can raise an error where it shouldn't. For example, `text_response.replace(status=200).text` can cause a false alarm from mypy.